packages/perseus/src/widgets/interactive-graphs/graphs/use-draggable.ts

```diff
@@ -61,6 +61,17 @@ export function useDraggable(args: Params): DragState {
             const isKeyboard = type.includes("key");
             if (isKeyboard) {
                 event?.preventDefault();
+
+                // When a key is held down, we see multiple "keydown" events,
+                // followed by a single "keyup" event.
+                // For a single keypress, we only see a "keydown" event, no
+                // "keyup".
+                // We never want to process the keyup event as an intent to
+                // move so we bail on further processing here.
+                if (type === "keyup") {
+                    return;
+                }
```

Comment:
I bet this `keyup` weirdness was the cause of [this mysterious bug](https://khanacademy.atlassian.net/browse/LEMS-1934). I believe @nishasy has the hardware to test whether the bug is fixed or not 🙏 

The same bug probably exists in upstream Mafs, so maybe we should submit the fix there.

Reply:
I tested with an iPhone 13 and a Bluetooth Magic Keyboard. It seems to be fixed. I've asked SarahP to retest. If it works, I'll mark that bug as fixed! :) 

I'll also open a PR against Mafs as you suggest.

Reply:
@jeremywiebe Does the bug occur in prod/main with your keyboard setup?

Reply:
Yes. Here's video where I'm pressing the arrow key once every few seconds. You can see the double-movement each time. 

https://github.com/user-attachments/assets/c5e433b5-0f3d-41fa-98ed-8ef251985380